tests/functional/test_sssd.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Test sssd functionality."""
+
+import unittest
+
+from sssd import Sssd
+
+
+class TestGlauth(unittest.TestCase):
```

Comment:
I think you mean `TestSSSDCharm` here :wink: 

```suggestion
class TestSSSDCharm(unittest.TestCase):
```

Reply:
Oh jeez, yup haha